features/autoconsent.json

```diff
@@ -21,6 +21,21 @@
     }, {
         "domain": "zeitraum-moebel.de",
         "reason": "broken autoconsent rule"
+    }, {
+        "domain": "mathebibel.de",
```

Comment:
Are these all considered critical breakage according to our ratings?

Reply:
Yeah, scrolling is not working for ~30s on a first visit :(